internal/provider/data_source_fmc_device_ha_pairs_test.go

```diff
@@ -0,0 +1,155 @@
+	checks = append(checks, resource.TestCheckResourceAttr("data.fmc_device_ha_pairs.test", "stateful_failover_interface_id", "<interface uuid>"))
+	checks = append(checks, resource.TestCheckResourceAttr("data.fmc_device_ha_pairs.test", "stateful_failover_interface_type", "PhysicalInterface"))
+	checks = append(checks, resource.TestCheckResourceAttr("data.fmc_device_ha_pairs.test", "action", "SWITCH"))
+	checks = append(checks, resource.TestCheckResourceAttr("data.fmc_device_ha_pairs.test", "force_break", "false"))
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceFmcDeviceHAPairsConfig(),
+				Check:  resource.ComposeTestCheckFunc(checks...),
+			},
+			{
+				Config: testAccNamedDataSourceFmcDeviceHAPairsConfig(),
+				Check:  resource.ComposeTestCheckFunc(checks...),
+			},
+		},
+	})
+}
+
+// End of section. //template:end testAccDataSource
+
+// Section below is generated&owned by "gen/generator.go". //template:begin testPrerequisites
+// End of section. //template:end testPrerequisites
+
+// Section below is generated&owned by "gen/generator.go". //template:begin testAccDataSourceConfig
+
+func testAccDataSourceFmcDeviceHAPairsConfig() string {
+	config := `resource "fmc_device_ha_pairs" "test" {` + "\n"
+	config += `	name = "FTD_HA"` + "\n"
+	config += `	primary_device_id = "<FTD1_ID>"` + "\n"
+	config += `	secondary_device_id = "<FTD2_ID>"` + "\n"
+	config += `	is_encryption_enabled = false` + "\n"
+	config += `	use_same_link_for_failovers = false` + "\n"
+	config += `	shared_key = "cisco123"` + "\n"
+	config += `	enc_key_generation_scheme = "CUSTOM"` + "\n"
+	config += `	lan_failover_standby_ip = "1.1.1.2"` + "\n"
+	config += `	lan_failover_active_ip = "1.1.1.1"` + "\n"
+	config += `	lan_failover_name = "LAN-INTERFACE"` + "\n"
+	config += `	lan_failover_subnet_mask = "255.255.255.0"` + "\n"
+	config += `	lan_failover_ipv6_addr = false` + "\n"
+	config += `	lan_failover_interface_name = "GigabitEthernet0/0"` + "\n"
+	config += `	lan_failover_interface_id = "<interface uuid>"` + "\n"
+	config += `	lan_failover_interface_type = "PhysicalInterface"` + "\n"
+	config += `	stateful_failover_standby_ip = "1.1.1.2"` + "\n"
+	config += `	stateful_failover_active_ip = "1.1.1.1"` + "\n"
+	config += `	stateful_failover_name = "Stateful-INTERFACE"` + "\n"
+	config += `	stateful_failover_subnet_mask = "255.255.255.0"` + "\n"
+	config += `	stateful_failover_ipv6_addr = false` + "\n"
+	config += `	stateful_failover_interface_name = "GigabitEthernet0/0"` + "\n"
+	config += `	stateful_failover_interface_id = "<interface uuid>"` + "\n"
+	config += `	stateful_failover_interface_type = "PhysicalInterface"` + "\n"
+	config += `	action = "SWITCH"` + "\n"
+	config += `	force_break = false` + "\n"
```

Comment:
I don't think this option will help create HA pair. Options can be excluded from tests using `exclude_test` yaml parameter.

Reply:
done